server/routes/charts.js

```diff
@@ -151,9 +163,14 @@ router
       const chart = await dataDb
         .collection(collections.charts)
         .findOne(
-          { _id: new ObjectID(req.params.chart_id) },
+          { _id: new ObjectID(req.params.chart_id), owner: req.user },
           { projection: { _id: 0 } }
         )
+
+      if (chart)
+        return res
+          .status(403)
+          .json({ message: 'Not authorized. Please contact the admin' })
```

Comment:
should this response be returned if there is no chart? Right now it looks like it will do this if it finds a chart. Also, can we redirect to the charts list page in this case?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/135/commits/0f1c63d004fcc0ce722558e1b2dab472de9388c3